src/cookies.ts

```diff
@@ -46,6 +46,16 @@ export const cookies: {
     },
     key: 'da4',
   },
+  onboarding: {
+    opts: {
+      maxAge: 1000 * 60 * 30,
+      httpOnly: true,
+      signed: false,
+      secure: env === 'production',
```

Comment:
Don't we have a isProd function for this?
Otherwise the enum might be better we have somewhere

Reply:
Updated, thanks!